src/engine/Loader.ts

```diff
@@ -262,91 +260,36 @@ export class Loader extends Class implements CanLoad {
     }
   }
 
+  update(_engine: Engine, _delta: number): void {
+    // override me
+  }
+
+  data: Loadable<any>[];
+
   /**
    * Begin loading all of the supplied resources, returning a promise
    * that resolves when loading of all is complete
    */
-  public load(): Promise<any> {
-    const complete = new Promise<any>((resolve) => {
-      const me = this;
-      if (this._resourceList.length === 0) {
-        me.showPlayButton().then(() => {
-          // Unlock audio context in chrome after user gesture
-          // https://github.com/excaliburjs/Excalibur/issues/262
-          // https://github.com/excaliburjs/Excalibur/issues/1031
-          WebAudio.unlock().then(() => {
-            me.hidePlayButton();
-            me.oncomplete.call(me);
-            resolve();
-          });
-        });
-        return;
-      }
+  public async load(): Promise<Loadable<any>[]> {
+    await Promise.all(this._resourceList.map(r => r.load().finally(() => {
+      // capture progress
+      this._numLoaded++;
+    })));
+
+    // short delay in showing the button for aesthetics
+    await delay(200);
+    await this.showPlayButton();
+    // Unlock audio context in chrome after user gesture
```

Comment:
It technically works everywhere, but chrome is the reason for it originally

Reply:
Should I amend the comment to remove the "chrome" specific reference?

Reply:
If it's relevant for all browsers, it may be appropriate to remove the reference. If it is a Chrome-specific edge case, capitalizing it should be enough to make the context clear.